wsagent/che-core-api-git/src/main/java/org/eclipse/che/api/git/GitUrl.java

```diff
@@ -8,7 +8,7 @@
  * Contributors:
  *   Codenvy, S.A. - initial API and implementation
  *******************************************************************************/
-package org.eclipse.che.git.impl.nativegit;
+package org.eclipse.che.api.git;
```

Comment:
removed the `getCodenvyTimeStampKeyLabel` method from the GitUrl class and renamed GitUrl -> GitUrlUtils